eth/handler.go

```diff
@@ -112,7 +112,7 @@ func NewProtocolManager(config *core.ChainConfig, singleMiner bool, networkId in
 		quitSync:    make(chan struct{}),
 		raftMode:    raftMode,
 	}
-	if singleMiner {
+	if assumeSyncedInitially {
 		manager.synced = uint32(1)
```

Comment:
There are problems propagating txs from a non-blockmaker node, even if the flag `--singleblockmaker` is on. The temporal solution is implementing the patch indicated below:

https://github.com/ethereum/go-ethereum/issues/2769#issuecomment-242141562